core/edit_text_file/edit_text_file_list.talon-list

```diff
@@ -13,3 +13,5 @@ unix utilities: tags/terminal/unix_utility.talon-list
 abbreviations: settings/abbreviations.csv
 file extensions: settings/file_extensions.csv
 words to replace: settings/words_to_replace.csv
+contacts: settings/contacts.json
```

Comment:
This should probably be contacts json to make it really clear which one you are using

Reply:
Good call; done.